onadata/apps/logger/models/submission_counter.py

```diff
@@ -0,0 +1,16 @@
+import datetime
```

Comment:
I think it's not needed in python3 anymore, but I usually add `# coding: utf-8` on the first line to define the encoding.
I would keep it to be consistent with old code.

Reply:
Resolving because it looks like you took care of it 😄